README.md

```diff
@@ -44,6 +44,8 @@ actual content by running this command from the project directory:
 git lfs checkout
 ```
 
+To Run the tests, right click on the "proknow-sdk-test" folder and click on `Run Tests` option.

```

Comment:
I also wonder if it would be better to add this below the `Access to ProKnow` session since we also need that part to run the tests. 

```suggestion
To run the tests, right click on the "proknow-sdk-test" folder and click on `Run Tests` option.

```